index.js

```diff
@@ -0,0 +1,9 @@
+import Header from "./src/components/Header.js";
+import Timer from "./src/components/Timer.js";
+
+window.addEventListener(`load`, () => {
+  const header = new Header();
+  header.render();
+  const timer = new Timer();
+  timer.render();
+});
```

Comment:
```suggestion
import Header from "./src/components/Header.js";
import Timer from "./src/components/Timer.js";

const header = new Header();
header.render();
const timer = new Timer();
timer.render();
```
running code on load event is not necessary when we use `script` tag with `type="module"`

Reply:
branch updated